public/schema/JobTemplates-01.yaml

```diff
@@ -6,6 +6,10 @@ additionalProperties: false
 required:
 - products
 - scenarios
+patternProperties:
+  "^.[a-z0-9_]+$":
```

Comment:
I didn't spot this, but this must be:
```
 '^\.[a-z0-9_]+$':
```
instead. Because the dot in the regex stands for anything.

I use single quotes because that's the recommened default quoting style.
If we used double quotes, we would have to escape the backslash too:
```
 "^\\.[a-z0-9_]+$":
```

Reply:
But what do you consider wrong with anything as first character, only the "user might select names we consider reserved later"?

Reply:
"user might select names we consider reserved later" is the main reason.
But also it could be confusing. If people use "somekey" and other people copy from it, it might make people believe that this is some standard.

Reply:
agreed

Reply:
Apparently fixed by #3104 - thanks @perlpunk for catching that!